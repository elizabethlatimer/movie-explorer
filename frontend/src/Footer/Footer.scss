.photoAttribution {
    position: absolute;
    height: 75px;
    bottom:0;
    width: 100%;
    background-color: rgba(54, 52, 52, 1);
    color: #fff;
    padding: 10px;
    font-size: 14px;
    p {
        margin: 5px 0px 0px 0px;
    }
    img {
        max-width: 50px;
        min-width: 50px;
        position: absolute;
        left: 40px;
        z-index: 1;

    }
}