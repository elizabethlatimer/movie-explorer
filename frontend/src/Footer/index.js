import React from 'react';
import './Footer.scss';

function Footer() {
  return (
    <div className="photoAttribution">
      <p>Background Photo by Jeremy Yap on Unsplash</p>
      <div><img className="TMDBlogo" src='https://www.themoviedb.org/assets/2/v4/logos/v2/blue_square_2-d537fb228cf3ded904ef09b136fe3fec72548ebc1fea3fbbd1ad9e36364db38b.svg'
        alt="The Movie Database Logo" /> This app uses the TMDb API but is not endorsed or certified by TMDb.</div>
    </div>
  )
}

export default Footer;