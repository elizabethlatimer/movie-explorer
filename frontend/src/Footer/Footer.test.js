import React from 'react';
import { render } from "@testing-library/react";

import Footer from './index';

it('renders without crashing', () => {
  render(<Footer />);
});

it('matches the snapshot', () => {
  const { asFragment } = render(<Footer />);
  expect(asFragment()).toMatchSnapshot();
});