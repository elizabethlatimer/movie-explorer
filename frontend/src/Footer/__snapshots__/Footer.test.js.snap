// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the snapshot 1`] = `
<DocumentFragment>
  <div
    class="photoAttribution"
  >
    <p>
      Background Photo by Jeremy Yap on Unsplash
    </p>
    <div
      class="api-attr"
    >
      <img
        alt="The Movie Database Logo"
        class="TMDBlogo"
        src="https://www.themoviedb.org/assets/2/v4/logos/v2/blue_square_2-d537fb228cf3ded904ef09b136fe3fec72548ebc1fea3fbbd1ad9e36364db38b.svg"
      />
      This app uses the TMDb API but is not endorsed or certified by TMDb.
    </div>
  </div>
</DocumentFragment>
`;
